Description: Strip embeded libraries from Rust by Examples book
 This patch removes embeded JS and CSS libraries from Rust by Examples e-book
 that comes bundled with the documentation
Author: Zixing Liu <zixing.liu@canonical.com>
Forwarded: not-needed
Last-Update: 2024-02-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git a/src/doc/rust-by-example/theme/index.hbs b/src/doc/rust-by-example/theme/index.hbs
index 1ae579f391..aa948ec2e7 100644
--- a/src/doc/rust-by-example/theme/index.hbs
+++ b/src/doc/rust-by-example/theme/index.hbs
@@ -33,10 +33,7 @@
         {{/if}}
 
         <!-- Fonts -->
-        <link rel="stylesheet" href="{{ path_to_root }}FontAwesome/css/font-awesome.css">
-        {{#if copy_fonts}}
-        <link rel="stylesheet" href="{{ path_to_root }}fonts/fonts.css">
-        {{/if}}
+        <link rel="stylesheet" href="{{ path_to_root }}css/font-awesome.min.css">
 
         <!-- Highlight.js Stylesheets -->
         <link rel="stylesheet" href="{{ path_to_root }}highlight.css">
